### PR TITLE
updated plan sharing form according to #2111, also added text to the …

### PR DESCRIPTION
**Pull request by xsrust**
…email field to clarify what happens if the user is not in our system

### Email
Screenshot 2019-07-16 at 12 23 56

###  Radio group
Screenshot 2019-07-16 at 12 24 03

### CODE REVIEW

**Comment by xsrust** [comment]
Rebased off development to try to get the travis to ignore the CVE in the bundle audit.

(Would have thought the pr part of the CI would do that automatically when it merged the new code in to development)

File: .gitignore [hunk 1]
```diff
@@ -53,7 +53,7 @@ config/branding.yml
 
 # Ignore some of the initializers
 config/initializers/recaptcha.rb
-# config/initializers/devise.rb
+config/initializers/devise.rb
```
Comment (briri) [inline .gitignore]: wow. thanks for catching this.

File: app/views/plans/_share_form.html.erb [hunk 2]
```diff
@@ -88,42 +88,51 @@
...
       <%= f.hidden_field :plan_id %>
       <%= f.fields_for :user do |user| %>
-        <%= user.label :email, _('Email'), class: 'control-label' %>
-        <%= user.email_field :email, for: :user, name: "user", class: "form-control", "aria-required": true %>
+        <%= user.label :email, _('Email'), class: 'control-label'%>
+        <%= user.email_field :email, for: :user, name: "user", class: "form-control",
+                            "aria-required": true,
+                            'data-toggle': 'tooltip',
+                            'data-html': true,
+                            title: _('Enter the email address of your collaborator: <ul><li> If they are already using DMPonline, they will see this plan on their dashboard, and recieve an email. </li> <li> If they are not currently using DMPonline, they will recieve an email inviting them to the tool so they can collaborate on your plan.</li></ul>') %>
```
Comment (briri) [inline app/views/plans/_share_form.html.erb]: Use the app name defined in the branding.yml here instead of 'DMPonline'.

File: .gitignore (hunk 1 above)
Comment (xsrust) [reply .gitignore]: Yeah, it kept overwriting my local version when I pulled a new branch down

File: app/views/plans/_share_form.html.erb (hunk 2 above)
Comment (xsrust) [reply app/views/plans/_share_form.html.erb]: Good catch,Good catch, completely slipped my mind, 
I'll update

**Pull request merged**